components/menu.js

```diff
@@ -1,42 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Link from 'next/link';
 import styles from './menu.module.css';
-export default function Menu({openMenu, setOpenMenu}){
 
-  const CloseMenu = () => {
-    setOpenMenu(false)
-  };
+export default function Menu(){
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const handleMenuClick = () => {
+    setIsMenuOpen(prevState => !prevState);
+  }
+
+  const hamburgerMenu = () => {
+    return !isMenuOpen ? 
+      (<div className={styles.hamburger} onClick={handleMenuClick}>
+        <div className={styles.hamburger_line}></div>
+      </div>)
+      :  
+      (<div className={styles.hamburger} onClick={handleMenuClick}>
+        <div className={styles.cross_line}></div>
+      </div>)
+  }
```

Comment:
Placed hamburger menu icon and its expanding menu in here `menu` component.

Created hamburger menu icon and cross icon with CSS instead of inserting the image.  I guess this could be lighter and increase loading speed.